docs/develop/modules/injective/exchange/04_state_transitions.md

```
@@ -445,3 +446,24 @@ Derivative market param update is handled by `DerivativeMarketParamUpdateProposa
+
+- Check if an existing grant from the granter already exists for the grantee
+- Calculate the new total amount of stake granted to the grantee by subtracting the existing grant amounts and adding the new grant amounts (essentially overwrites the existing grant amounts with the new grant amounts)
+- Ensure valid grant authorizations by making sure total amount granted is less than or equal to total amount staked by granter
+- Update grant amounts for grantee
+- Set grant to active if the current active grant is from the same granter or if there is no current active grant
+- Emit `EventGrantAuthorizations` with granter and grants
+
+
+## Stake Grant Activation
+
+**Steps**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)